torchbiggraph/converters/export_to_tsv.py

```diff
@@ -24,7 +24,8 @@ def make_tsv(
     checkpoint: str,
     relation_types: List[str],
     entities_by_type: Dict[str, List[str]],
-    output_file: IO[str],
+    ent_out_file: IO[str],
+    rel_out_file: IO[str],
```

Comment:
Similarly to the above, can you rename this `entities_output_file` and `relation_types_output_file`?